test/integration/project-management.spec.ts

```diff
@@ -234,13 +263,27 @@ function generateUniqueAppDisplayName() {
   return APP_DISPLAY_NAME_PREFIX + generateRandomString(APP_DISPLAY_NAME_SUFFIX_LENGTH);
 }
 
+/**
+ * @return {string} string that can be used as a unique project display name.
+ */
+function generateUniqueProjectDisplayName() {
+  return PROJECT_DISPLAY_NAME_PREFIX + generateRandomString(PROJECT_DISPLAY_NAME_SUFFIX_LENGTH);
+}
+
 /**
  * @return {boolean} True if the specified appNamespace belongs to these integration tests.
  */
 function isIntegrationTestApp(appNamespace: string): boolean {
   return (appNamespace.indexOf(APP_NAMESPACE_PREFIX) > -1);
 }
 
+/**
+ * @return {boolean} True if the specified appDisplayName belongs to these integration tests.
+ */
+function isIntegrationTestAppDisplayName(appDisplayName: string): boolean {
+  return appDisplayName && (appDisplayName.indexOf(APP_DISPLAY_NAME_PREFIX) > -1);
```

Comment:
Can we do a `appDisplayName.startsWith()` here?

Reply:
Done